
<!DOCTYPE html>

<html>
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" /><meta name="generator" content="Docutils 0.17.1: http://docutils.sourceforge.net/" />

    <title>Chapter 2 - Biology basics and measuring biology &#8212; Computational Longevity</title>
    
  <link href="_static/css/theme.css" rel="stylesheet">
  <link href="_static/css/index.ff1ffe594081f20da1ef19478df9384b.css" rel="stylesheet">

    
  <link rel="stylesheet"
    href="_static/vendor/fontawesome/5.13.0/css/all.min.css">
  <link rel="preload" as="font" type="font/woff2" crossorigin
    href="_static/vendor/fontawesome/5.13.0/webfonts/fa-solid-900.woff2">
  <link rel="preload" as="font" type="font/woff2" crossorigin
    href="_static/vendor/fontawesome/5.13.0/webfonts/fa-brands-400.woff2">

    
      

    
    <link rel="stylesheet" type="text/css" href="_static/pygments.css" />
    <link rel="stylesheet" type="text/css" href="_static/sphinx-book-theme.css?digest=c3fdc42140077d1ad13ad2f1588a4309" />
    <link rel="stylesheet" type="text/css" href="_static/togglebutton.css" />
    <link rel="stylesheet" type="text/css" href="_static/copybutton.css" />
    <link rel="stylesheet" type="text/css" href="_static/mystnb.css" />
    <link rel="stylesheet" type="text/css" href="_static/sphinx-thebe.css" />
    <link rel="stylesheet" type="text/css" href="_static/panels-main.c949a650a448cc0ae9fd3441c0e17fb0.css" />
    <link rel="stylesheet" type="text/css" href="_static/panels-variables.06eb56fa6e07937060861dad626602ad.css" />
    
  <link rel="preload" as="script" href="_static/js/index.be7d3bbb2ef33a8344ce.js">

    <script data-url_root="./" id="documentation_options" src="_static/documentation_options.js"></script>
    <script src="_static/jquery.js"></script>
    <script src="_static/underscore.js"></script>
    <script src="_static/doctools.js"></script>
    <script src="_static/clipboard.min.js"></script>
    <script src="_static/copybutton.js"></script>
    <script>let toggleHintShow = 'Click to show';</script>
    <script>let toggleHintHide = 'Click to hide';</script>
    <script>let toggleOpenOnPrint = 'true';</script>
    <script src="_static/togglebutton.js"></script>
    <script>var togglebuttonSelector = '.toggle, .admonition.dropdown, .tag_hide_input div.cell_input, .tag_hide-input div.cell_input, .tag_hide_output div.cell_output, .tag_hide-output div.cell_output, .tag_hide_cell.cell, .tag_hide-cell.cell';</script>
    <script src="_static/sphinx-book-theme.d59cb220de22ca1c485ebbdc042f0030.js"></script>
    <script async="async" src="https://unpkg.com/thebe@0.5.1/lib/index.js"></script>
    <script>
        const thebe_selector = ".thebe"
        const thebe_selector_input = "pre"
        const thebe_selector_output = ".output"
    </script>
    <script async="async" src="_static/sphinx-thebe.js"></script>
    <link rel="index" title="Index" href="genindex.html" />
    <link rel="search" title="Search" href="search.html" />
    <link rel="next" title="Chapter 3 - Hallmarks and biomarkers of aging" href="Chapter%203%20-%20Hallmarks%20and%20biomarkers%20of%20aging.html" />
    <link rel="prev" title="Chapter 1 - Motivation" href="Chapter%201%20-%20Motivation.html" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <meta name="docsearch:language" content="None">
    

    <!-- Google Analytics -->
    
  </head>
  <body data-spy="scroll" data-target="#bd-toc-nav" data-offset="80">
    
    <div class="container-fluid" id="banner"></div>

    

    <div class="container-xl">
      <div class="row">
          
<div class="col-12 col-md-3 bd-sidebar site-navigation show" id="site-navigation">
    
        <div class="navbar-brand-box">
    <a class="navbar-brand text-wrap" href="index.html">
      
      
      
      <h1 class="site-logo" id="site-title">Computational Longevity</h1>
      
    </a>
</div><form class="bd-search d-flex align-items-center" action="search.html" method="get">
  <i class="icon fas fa-search"></i>
  <input type="search" class="form-control" name="q" id="search-input" placeholder="Search this book..." aria-label="Search this book..." autocomplete="off" >
</form><nav class="bd-links" id="bd-docs-nav" aria-label="Main">
    <div class="bd-toc-item active">
        <ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="Chapter%201%20-%20Motivation.html">
   Chapter 1 - Motivation
  </a>
 </li>
</ul>
<ul class="current nav bd-sidenav">
 <li class="toctree-l1 current active">
  <a class="current reference internal" href="#">
   Chapter 2 - Biology basics and measuring biology
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="Chapter%203%20-%20Hallmarks%20and%20biomarkers%20of%20aging.html">
   Chapter 3 - Hallmarks and biomarkers of aging
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="Chapter%204%20-%20Longevity%20interventions%20and%20models.html">
   Chapter 4 - Longevity interventions and models
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="Chapter%205%20-%20References.html">
   References
  </a>
 </li>
</ul>

    </div>
</nav> <!-- To handle the deprecated key -->

<div class="navbar_extra_footer">
  Powered by <a href="https://jupyterbook.org">Jupyter Book</a>
</div>

</div>


          


          
<main class="col py-md-3 pl-md-4 bd-content overflow-auto" role="main">
    
    <div class="topbar container-xl fixed-top">
    <div class="topbar-contents row">
        <div class="col-12 col-md-3 bd-topbar-whitespace site-navigation show"></div>
        <div class="col pl-md-4 topbar-main">
            
            <button id="navbar-toggler" class="navbar-toggler ml-0" type="button" data-toggle="collapse"
                data-toggle="tooltip" data-placement="bottom" data-target=".site-navigation" aria-controls="navbar-menu"
                aria-expanded="true" aria-label="Toggle navigation" aria-controls="site-navigation"
                title="Toggle navigation" data-toggle="tooltip" data-placement="left">
                <i class="fas fa-bars"></i>
                <i class="fas fa-arrow-left"></i>
                <i class="fas fa-arrow-up"></i>
            </button>
            
            
<div class="dropdown-buttons-trigger">
    <button id="dropdown-buttons-trigger" class="btn btn-secondary topbarbtn" aria-label="Download this page"><i
            class="fas fa-download"></i></button>

    <div class="dropdown-buttons">
        <!-- ipynb file if we had a myst markdown file -->
        
        <!-- Download raw file -->
        <a class="dropdown-buttons" href="_sources/Chapter 2 - Biology basics and measuring biology.md"><button type="button"
                class="btn btn-secondary topbarbtn" title="Download source file" data-toggle="tooltip"
                data-placement="left">.md</button></a>
        <!-- Download PDF via print -->
        <button type="button" id="download-print" class="btn btn-secondary topbarbtn" title="Print to PDF"
                onclick="printPdf(this)" data-toggle="tooltip" data-placement="left">.pdf</button>
    </div>
</div>

            <!-- Source interaction buttons -->

            <!-- Full screen (wrap in <a> to have style consistency -->

<a class="full-screen-button"><button type="button" class="btn btn-secondary topbarbtn" data-toggle="tooltip"
        data-placement="bottom" onclick="toggleFullScreen()" aria-label="Fullscreen mode"
        title="Fullscreen mode"><i
            class="fas fa-expand"></i></button></a>

            <!-- Launch buttons -->

        </div>

        <!-- Table of contents -->
        <div class="d-none d-md-block col-md-2 bd-toc show noprint">
            
            <div class="tocsection onthispage pt-5 pb-3">
                <i class="fas fa-list"></i> Contents
            </div>
            <nav id="bd-toc-nav" aria-label="Page">
                <ul class="visible nav section-nav flex-column">
 <li class="toc-h1 nav-item toc-entry">
  <a class="reference internal nav-link" href="#">
   Chapter 2 - Biology basics and measuring biology
  </a>
  <ul class="visible nav section-nav flex-column">
   <li class="toc-h2 nav-item toc-entry">
    <a class="reference internal nav-link" href="#genes">
     Genes
    </a>
   </li>
   <li class="toc-h2 nav-item toc-entry">
    <a class="reference internal nav-link" href="#transcription">
     Transcription
    </a>
   </li>
   <li class="toc-h2 nav-item toc-entry">
    <a class="reference internal nav-link" href="#translation">
     Translation
    </a>
   </li>
   <li class="toc-h2 nav-item toc-entry">
    <a class="reference internal nav-link" href="#gene-expression-regulation">
     Gene expression regulation
    </a>
   </li>
   <li class="toc-h2 nav-item toc-entry">
    <a class="reference internal nav-link" href="#epigenetics">
     Epigenetics
    </a>
   </li>
   <li class="toc-h2 nav-item toc-entry">
    <a class="reference internal nav-link" href="#chromosomes-telomeres-and-telomerase">
     Chromosomes telomeres and telomerase
    </a>
   </li>
   <li class="toc-h2 nav-item toc-entry">
    <a class="reference internal nav-link" href="#mitochondria-and-oxidative-stress-theory">
     Mitochondria and oxidative stress theory
    </a>
   </li>
   <li class="toc-h2 nav-item toc-entry">
    <a class="reference internal nav-link" href="#cellular-senescence">
     Cellular senescence
    </a>
   </li>
   <li class="toc-h2 nav-item toc-entry">
    <a class="reference internal nav-link" href="#autophagy">
     Autophagy
    </a>
   </li>
   <li class="toc-h2 nav-item toc-entry">
    <a class="reference internal nav-link" href="#inflammation">
     Inflammation
    </a>
   </li>
  </ul>
 </li>
 <li class="toc-h1 nav-item toc-entry">
  <a class="reference internal nav-link" href="#quantifying-biological-research-in-longevity">
   Quantifying biological research in longevity
  </a>
  <ul class="visible nav section-nav flex-column">
   <li class="toc-h2 nav-item toc-entry">
    <a class="reference internal nav-link" href="#genomics-data-for-longevity">
     Genomics data for longevity
    </a>
   </li>
   <li class="toc-h2 nav-item toc-entry">
    <a class="reference internal nav-link" href="#epigenomics-data-for-longevity">
     Epigenomics data for longevity
    </a>
   </li>
   <li class="toc-h2 nav-item toc-entry">
    <a class="reference internal nav-link" href="#transcriptomics-data-for-longevity">
     Transcriptomics data for longevity
    </a>
   </li>
   <li class="toc-h2 nav-item toc-entry">
    <a class="reference internal nav-link" href="#proteomics-data-for-longevity">
     Proteomics data for longevity
    </a>
   </li>
   <li class="toc-h2 nav-item toc-entry">
    <a class="reference internal nav-link" href="#metabolomics-data-for-longevity">
     Metabolomics data for longevity
    </a>
   </li>
   <li class="toc-h2 nav-item toc-entry">
    <a class="reference internal nav-link" href="#multi-omics-approach-in-longevity">
     Multi -omics approach in longevity
    </a>
   </li>
   <li class="toc-h2 nav-item toc-entry">
    <a class="reference internal nav-link" href="#network-biology-approach-in-longevity">
     Network biology approach in longevity
    </a>
   </li>
   <li class="toc-h2 nav-item toc-entry">
    <a class="reference internal nav-link" href="#general-measurament-tools">
     General Measurament tools
    </a>
    <ul class="nav section-nav flex-column">
     <li class="toc-h3 nav-item toc-entry">
      <a class="reference internal nav-link" href="#sequencing">
       Sequencing
      </a>
     </li>
     <li class="toc-h3 nav-item toc-entry">
      <a class="reference internal nav-link" href="#qpcr">
       qPCR
      </a>
     </li>
     <li class="toc-h3 nav-item toc-entry">
      <a class="reference internal nav-link" href="#antibody-based-methods">
       Antibody-based Methods
      </a>
     </li>
     <li class="toc-h3 nav-item toc-entry">
      <a class="reference internal nav-link" href="#x-ray-crystallography-and-cryo-em">
       X-Ray Crystallography and Cryo-EM
      </a>
     </li>
    </ul>
   </li>
   <li class="toc-h2 nav-item toc-entry">
    <a class="reference internal nav-link" href="#data-sources">
     Data sources
    </a>
    <ul class="nav section-nav flex-column">
     <li class="toc-h3 nav-item toc-entry">
      <a class="reference internal nav-link" href="#data-sources-specifically-dedicated-to-aging">
       Data sources specifically dedicated to aging
      </a>
     </li>
    </ul>
   </li>
  </ul>
 </li>
</ul>

            </nav>
        </div>
    </div>
</div>
    <div id="main-content" class="row">
        <div class="col-12 col-md-9 pl-md-3 pr-md-0">
            <!-- Table of contents that is only displayed when printing the page -->
            <div id="jb-print-docs-body" class="onlyprint">
                <h1>Chapter 2 - Biology basics and measuring biology</h1>
                <!-- Table of contents -->
                <div id="print-main-content">
                    <div id="jb-print-toc">
                        
                        <div>
                            <h2> Contents </h2>
                        </div>
                        <nav aria-label="Page">
                            <ul class="visible nav section-nav flex-column">
 <li class="toc-h1 nav-item toc-entry">
  <a class="reference internal nav-link" href="#">
   Chapter 2 - Biology basics and measuring biology
  </a>
  <ul class="visible nav section-nav flex-column">
   <li class="toc-h2 nav-item toc-entry">
    <a class="reference internal nav-link" href="#genes">
     Genes
    </a>
   </li>
   <li class="toc-h2 nav-item toc-entry">
    <a class="reference internal nav-link" href="#transcription">
     Transcription
    </a>
   </li>
   <li class="toc-h2 nav-item toc-entry">
    <a class="reference internal nav-link" href="#translation">
     Translation
    </a>
   </li>
   <li class="toc-h2 nav-item toc-entry">
    <a class="reference internal nav-link" href="#gene-expression-regulation">
     Gene expression regulation
    </a>
   </li>
   <li class="toc-h2 nav-item toc-entry">
    <a class="reference internal nav-link" href="#epigenetics">
     Epigenetics
    </a>
   </li>
   <li class="toc-h2 nav-item toc-entry">
    <a class="reference internal nav-link" href="#chromosomes-telomeres-and-telomerase">
     Chromosomes telomeres and telomerase
    </a>
   </li>
   <li class="toc-h2 nav-item toc-entry">
    <a class="reference internal nav-link" href="#mitochondria-and-oxidative-stress-theory">
     Mitochondria and oxidative stress theory
    </a>
   </li>
   <li class="toc-h2 nav-item toc-entry">
    <a class="reference internal nav-link" href="#cellular-senescence">
     Cellular senescence
    </a>
   </li>
   <li class="toc-h2 nav-item toc-entry">
    <a class="reference internal nav-link" href="#autophagy">
     Autophagy
    </a>
   </li>
   <li class="toc-h2 nav-item toc-entry">
    <a class="reference internal nav-link" href="#inflammation">
     Inflammation
    </a>
   </li>
  </ul>
 </li>
 <li class="toc-h1 nav-item toc-entry">
  <a class="reference internal nav-link" href="#quantifying-biological-research-in-longevity">
   Quantifying biological research in longevity
  </a>
  <ul class="visible nav section-nav flex-column">
   <li class="toc-h2 nav-item toc-entry">
    <a class="reference internal nav-link" href="#genomics-data-for-longevity">
     Genomics data for longevity
    </a>
   </li>
   <li class="toc-h2 nav-item toc-entry">
    <a class="reference internal nav-link" href="#epigenomics-data-for-longevity">
     Epigenomics data for longevity
    </a>
   </li>
   <li class="toc-h2 nav-item toc-entry">
    <a class="reference internal nav-link" href="#transcriptomics-data-for-longevity">
     Transcriptomics data for longevity
    </a>
   </li>
   <li class="toc-h2 nav-item toc-entry">
    <a class="reference internal nav-link" href="#proteomics-data-for-longevity">
     Proteomics data for longevity
    </a>
   </li>
   <li class="toc-h2 nav-item toc-entry">
    <a class="reference internal nav-link" href="#metabolomics-data-for-longevity">
     Metabolomics data for longevity
    </a>
   </li>
   <li class="toc-h2 nav-item toc-entry">
    <a class="reference internal nav-link" href="#multi-omics-approach-in-longevity">
     Multi -omics approach in longevity
    </a>
   </li>
   <li class="toc-h2 nav-item toc-entry">
    <a class="reference internal nav-link" href="#network-biology-approach-in-longevity">
     Network biology approach in longevity
    </a>
   </li>
   <li class="toc-h2 nav-item toc-entry">
    <a class="reference internal nav-link" href="#general-measurament-tools">
     General Measurament tools
    </a>
    <ul class="nav section-nav flex-column">
     <li class="toc-h3 nav-item toc-entry">
      <a class="reference internal nav-link" href="#sequencing">
       Sequencing
      </a>
     </li>
     <li class="toc-h3 nav-item toc-entry">
      <a class="reference internal nav-link" href="#qpcr">
       qPCR
      </a>
     </li>
     <li class="toc-h3 nav-item toc-entry">
      <a class="reference internal nav-link" href="#antibody-based-methods">
       Antibody-based Methods
      </a>
     </li>
     <li class="toc-h3 nav-item toc-entry">
      <a class="reference internal nav-link" href="#x-ray-crystallography-and-cryo-em">
       X-Ray Crystallography and Cryo-EM
      </a>
     </li>
    </ul>
   </li>
   <li class="toc-h2 nav-item toc-entry">
    <a class="reference internal nav-link" href="#data-sources">
     Data sources
    </a>
    <ul class="nav section-nav flex-column">
     <li class="toc-h3 nav-item toc-entry">
      <a class="reference internal nav-link" href="#data-sources-specifically-dedicated-to-aging">
       Data sources specifically dedicated to aging
      </a>
     </li>
    </ul>
   </li>
  </ul>
 </li>
</ul>

                        </nav>
                    </div>
                </div>
            </div>
            
              <div>
                
  <section class="tex2jax_ignore mathjax_ignore" id="chapter-2-biology-basics-and-measuring-biology">
<h1>Chapter 2 - Biology basics and measuring biology<a class="headerlink" href="#chapter-2-biology-basics-and-measuring-biology" title="Permalink to this headline">¶</a></h1>
<p>Here I have set out to outline following things:</p>
<ul class="simple">
<li><p>Basics of (longevity) biology.</p></li>
<li><p>Quantifying biological research in longevity.</p></li>
<li><p>Network biology approach in longevity.</p></li>
<li><p>Tools for measuring the data.</p></li>
<li><p>Data sources specifically dedicated to aging.</p></li>
</ul>
<p>Cell, as a basic unit of life, contains answers to all of the life questions. Including longevity. Studying it is a good start to tackle longevity.</p>
<p><a class="reference external" href="https://www.nature.com/scitable/ebooks/cntNm-14749010/118237871/">Essentials of Cell Biology</a> provides a great introduction. Here I will try to briefly reiterate definitions that are relevant.</p>
<p><img alt="Cell" src="https://cdn.lecturio.com/assets/The-cell-and-cellular-organelles.jpg" /></p>
<ul class="simple">
<li><p>Proteins are responsible for structural, transport, signaling and storage roles. Additionally they are functional components of the cell, acting as enzymes — molecular machines that catalyze reactions.</p></li>
<li><p>Small molecules is a low molecular weight organic compound that may regulate a biological process, with a size on the order of 1 nm. Some examples are phenolic compounds, alkaloids and sugars.</p></li>
<li><p>Nucleic acids represent information, with DNA encoding genetic information and RNA playing multiple roles. Messenger RNA carries information from DNA to ribosomes. molecular machines partially constructed from RNA themselves, which synthesize proteins by translating nucleotides into amino acids.</p></li>
<li><p>Water, small molecules and inorganic ions.</p></li>
<li><p>Carbohydrates used for nutrients, energy storage and some cases, such as cellulose in plants, providing cell structure.</p></li>
<li><p>Lipids used for energy storage, signaling and cellular membranes.</p></li>
</ul>
<section id="genes">
<h2>Genes<a class="headerlink" href="#genes" title="Permalink to this headline">¶</a></h2>
<p><img alt="" src="https://www.news-medical.net/image.axd?picture=2020%2F6%2Fshutterstock_1238405779.jpg" /></p>
<p>From a structural perspective, DNA is a double-helix of two chains of nucleotides (adenine, thymine, guanine, cytosine, ATGC). If we take one of the strands of this double-helix, one can identify different sections that do different things, these are the genes. In turn, a given gene can be further broken down into codons, or triplets of nucleotides, e.g. the sequence A-T-G.</p>
</section>
<section id="transcription">
<h2>Transcription<a class="headerlink" href="#transcription" title="Permalink to this headline">¶</a></h2>
<p><img alt="" src="https://www.thoughtco.com/thmb/a41xHFS4GT-SugfXXNdWCKZD9UY=/768x0/filters:no_upscale():max_bytes(150000):strip_icc():format(webp)/protein_synthesis-114c6e97b3494f04abc60643d7fda11a.jpg" /></p>
<p>To go from the sequence of codons to something more interesting, the DNA is copied by the enzyme RNA polymerase into a strand of RNA (pre-mRNA) which is then modified to eventually become messenger RNA (mRNA). A single strand of pre-mRNA can yield many strands of mRNA. [TODO visualise exons and introns] A strand of pre-mRNA can be thought of as a chain of “useful” sections (exons) and “separator” sections (introns). pre-mRNA is cut at the introns and joined back together by a spliceosome, a complex association of 80 proteins conforming a molecular machine. This process is not always the same! Depending on how this cutting and pasting happens, different mRNA sequences will result, in turn leading to different proteins being made out of the same gene.</p>
</section>
<section id="translation">
<h2>Translation<a class="headerlink" href="#translation" title="Permalink to this headline">¶</a></h2>
<p>mRNA is basically like one of the two strands of DNA but instead of using ATGC, it has uracil instead of thymine, so AUGC. The strand of mRNA is sent into the cytoplasm of the cell where they reach the ribosomes. There, yet another form of RNA, transfer RNA (tRNA) [TODO visualise tRNA and reading mRNA and producing amino acids ] reads the mRNA and assembles a chain of amino acids in the order dictated by the codons. So if the mRNA says AUG-GCU-UAA we get methionine-alanine (The last codon is a stop codon, and stops the transcription right there).</p>
<p>But because there are so many of them, and because they have a few functional groups attached to them, they will interact with each other and end up curling up to conform a protein. Occasionally more than a single chain will be required for a protein to be assembled.</p>
<p>To recap, if we write down the DNA sequence, we can know what amino acids will be produced and how they will be stitched together. But knowing how the protein will assemble itself, and what final shape it will adopt is a very hard problem.</p>
<p>The narrative outlined above is what is known as the Central dogma of molecular biology The real picture is somewhat more complicated. For example, once a protein is floating around, this protein can interact with the DNA itself and regulate its activity; this is the basis for epigenetics.</p>
</section>
<section id="gene-expression-regulation">
<h2>Gene expression regulation<a class="headerlink" href="#gene-expression-regulation" title="Permalink to this headline">¶</a></h2>
<p>We still need one more thing: Almost every cell in an organism shares the same DNA (There are exceptions, as in the adaptive immune system), yet a neuron is markedly different from a cell in the liver. This is because not every gene is expressed in every cell: they can be switched on and off (gene regulation), by means of either internal (maybe the concentration of some protein, what proteins are present inside) or external (exposition to certain molecules outside of the cell) signals.</p>
<p>This regulation can happen at various levels in the gene -&gt; protein route explained above.</p>
<ul class="simple">
<li><p>Before transcription: This is covered under epigenetics, in the next section.</p></li>
<li><p>During transcription: Certain proteins, called transcription factors, can bind to specific genes and promote or repress their transcription into mRNA.</p></li>
<li><p>During translation: One can also interfere with the process that takes mRNA and assembles the amino acids together. For example, one can reduce the lifespan of mRNA in the cell, if there’s less of it, less proteins will be made. This is the basis of RNA interference (RNAi), used to knock down or reduce the activity of a gene.</p></li>
<li><p>Post-translation: Even when the route has come to an end and a protein has been assembled, one can still act on these newly assembled proteins, be it by disassembling them, or by tagging them by attaching certain chemical groups to them. An example of this is phosphorylation, where a phosphoryl group is attached to a protein. Phosphorylation - or any other modification to a protein - doesn’t always have the same effect, it may increase the activity of a protein, or reduce it, depending on what the protein is. Another ubiquitous mechanism is ubiquitination, sticking small ubiquitin proteins to other proteins; proteins so tagged are flagged for the cell to recycle them, reducing the concentration of the protein in the cytoplasm, leading to reduced activity.</p></li>
</ul>
</section>
<section id="epigenetics">
<h2>Epigenetics<a class="headerlink" href="#epigenetics" title="Permalink to this headline">¶</a></h2>
<p><img alt="" src="https://www.frontiersin.org/files/Articles/613636/fgene-11-613636-HTML-r2/image_m/fgene-11-613636-g001.jpg" /></p>
<p>So far we have considered DNA as something that’s just sitting there, like a file in a hard drive from which we only read. But it’s possible to interpret this genetic code in different ways, and this is the main mechanism by which cells end up being so different.</p>
<p>The DNA double helix is spun around histones, another kind of protein (To make things more convoluted, there are different kinds of histones). This wrapped DNA plus some other ancillary proteins form together the chromatin molecule. Both the nucleotides itself and the histones can have their chemical structure altered without having the sequence of nucleotides changed, and these changes affect what genes get expressed -this is like writing meta-data to our source file. In the case of DNA, one way is sticking methyl groups to the DNA nucleotides, which typically will stop the gene from even being transcribed. The histones that are part of chromatin can also be methylated, though here this mark can act both as a promoter or repressor of gene expression. Other forms of epigenetic modifications are possible, like (de)acetylation, similar to methylation but involving an acetyl group, which typically opens up the chromatin facilitating gene expression; we’ll see in a moment a class of proteins, the sirtuin, that are directly involved in this process and are relevant to the aging process.</p>
<p>Another mode of epigenetic regulation of gene expression is the formation of heterochromatin: DNA is not floating around with all of its nucleotides accessible to the transcription machinery; rather it sometimes curls in a tightly packed form -heterochromatin- that inhibits transcription.</p>
<p>It seems like the relevance of epigenetics to aging is higher than it was thought years ago. (Pal and Tyler (2016), Kane and Sinclair, (2019)), and David Sinclair in particular defends a theory of aging that puts it at the core of the issue.</p>
</section>
<section id="chromosomes-telomeres-and-telomerase">
<h2>Chromosomes telomeres and telomerase<a class="headerlink" href="#chromosomes-telomeres-and-telomerase" title="Permalink to this headline">¶</a></h2>
<p><img alt="" src="https://www.news-medical.net/image.axd?picture=2019%2F5%2Fshutterstock_710795275.jpg" /></p>
<p>One might think that inside the cellular nucleus there are the very visible and nicely shaped chromosomes that we see in textbooks. But this is not true: They are generally very hard to see as they are in an uncoiled state, facilitating their interaction with other proteins. It is only when the cell is about to divide that they become tightly packed.</p>
<p>At the end of the DNA molecule there are telomeres, sections of repeated nucleotides (In humans, TTAGGG). In turn at the end of the telomere there is a ‘knot’ or loop around the shelterin protein complex. What shelterin (Palm &amp; de Lange, 2008) does is to hide (Or shelter hah) the end of the telomere from the DNA repair machinery: DNA looks like a continuous strand for most of its length except at the ends, so it is efficient for the cell to recognise ‘ends’ of this strand as breaks, and try to repair them. But of course you wouldn’t want that in the end of the chromosome, because it’s a legitimate feature. However, if true damage happens to the telomeric section, it’s not going to be repaired, so telomeres steadily accumulate damage. If shelterin is removed, the cell recognises this as a large DNA damage point and commits cellular suicide (apoptosis) or enters replicative arrest (senescence). We’ll get to senescent cells later.</p>
<p>When the cell divides and chromosomes are assembled, telomeres end up as ‘caps’ in the arms of the chromosomes, but when the DNA is being replicated, because of the way the enzyme DNA polymerase (Which does the replication job) works, it fails to copy a few nucleotides at the end, so with each cell replication, the telomeres shorten by about 50 base pairs (bp).</p>
<p>Once the telomere is short enough, it won’t be able to maintain the ‘loop’ structure at its end, thus it will be recognised as a DNA break, triggering cell death. Thus given an initial telomeric length and a rate of attrition per cell division, one could calculate how many times a cell can replicate. This is known as the Hayflick limit.</p>
<p>It’s important to remark that rarely the cell will actually reach a point where there is no telomeres left at all: at birth, telomeres are around 10 kilobases long and every year we lose around 25 base pairs (Shammas, 2012) so in theory a cell has enough telomeres to go for 400 years, assuming that there is no other damage to telomeres! (Actually less than that, because there is a critically low telomere length, ~ 2 kb for humans at which the cell triggers cell death)</p>
<p>Cells however induce apoptosis or senescence way before it gets to this point, so while running out of telomere is the ultimate failsafe, it’s more common for the telomere to have accumulated so much damage that this suffices to trigger cell death. Thus telomeres act as instruments to measure damage in the cell in general (Victorelli &amp; Passos, 2017).</p>
<p>It’s important to note that cell death or senescence will happen when the shorter telomere is shorter than some threshold. For some time much attention was paid to average telomere length, but better measurement techniques made it clearer that it wasn’t that way.</p>
<p>However, we do have some cells that replicate a lot (Like the stem cells that produce red blood cells) and their telomeres are not long enough to live long enough through our lives. So for those cells that do need to replicate there is an reverse transcriptase (It makes DNA from RNA, while regular transcriptase does the opposite) enzyme, telomerase, that can lengthen the telomeres. Telomerase is made up of two core proteins, those encoded by the TERT and TERC (An RNA template) genes. The fact that telomerase exists is the reason why sometimes one may read that ‘X healthy habit lengthen telomeres’, technically it is possible for telomerase to lead to longer telomeres in the population of some cells like red blood cells or leukocytes (The ones typically used to measure telomere length). However, for the rest of the cells, telomere length can only ever shorten. (With the rare exception of the Alternative Lengthening of Telomeres (ALT) mechanism).</p>
<p>Telomeres are useful to suppress cancer: If a cell replicates too fast, it will exhaust their telomeres and die. So typically when cancer does happen it is because telomerase somehow has become active, or more rarely (10% of cases) due to ALT.</p>
<p>Some mutations can cause deficiencies in telomere length and accurate assays can help diagnose this to guide treatment, both in animal models and in humans, mutations in telomerase or abnormalities in the telomeres is linked to disease like pulmonary fibrosis or dyskeratosis congenita (Alder et al. 2018, Lopez-Otin et al., 2013) however for the general population telomere length does not seem very predictive of illness.</p>
</section>
<section id="mitochondria-and-oxidative-stress-theory">
<h2>Mitochondria and oxidative stress theory<a class="headerlink" href="#mitochondria-and-oxidative-stress-theory" title="Permalink to this headline">¶</a></h2>
<p><img alt="" src="https://www.genome.gov/sites/default/files/media/images/2021-11/mitochondria_0.jpg" /></p>
<p><strong>Reactive Oxygen Species (ROS)</strong></p>
<p>We’ve been talking about damage. One kind of such damage is ROS, the only known molecular species present in the organism capable of cutting off covalent bonds indiscriminately. ROS is produced in substantial amounts at mitochondria as a byproduct of ATP (a molecule used as energy transport) production. ROS can damage among other things telomeres, which leads up to cellular senescence and in turn, aging.</p>
<p>David Sinclair, in Lifespan claims that this is not an issue for aging and that this is the consensus in the field, referring to a theory of aging known as the Oxidative Stress Theory of Aging. Indeed, back in 2009 there was a paper (Perez et al. 2009) proclaiming the theory dead, on the grounds that antioxidant intake does not prolong life, and that deletion of genes that produce antioxidants does not reduce lifespan (Likewise, the addition of extra copies of antioxidant-generating genes did not lengthen lifespan). Likewise the same was said in 2010 of the Mitochondrial Free Radical Theory of Aging or mFRTA (Lapointe, 2010).</p>
<p>Other papers were coming out that were less pessimistic. Salmon et al (2010) admits that ROS may not be involved in lifespan, but that it may be in healthspan, or that the effects may be different (worse) if oxidative stress is chronic rather than acute (As it is in experiments).</p>
<p>In 2013, ROS was shown to be a minor cause of mutations and damage in mtDNA (Kennedy et al. 2013). But that same year, the proponents of the oxidative aging theory replied (Barja, 2013), arguing that the criticisms are unfounded: Antioxidant intake may not lengthen maximum lifespan, but it does increase mean lifespan, longer lived species generate less ROS, additionally the location of ROS generation (In the mitochondria) ensures that extra antioxidants won’t get there, so antioxidants may reduce damage outside but not damage to mtDNA. Nor does the theory need the assumption that ROS production increases with age: A constant rate of damage can cause an exponentially increasing mortality rate.</p>
<p>In 2014, an extensive literature review (Stuart et al. 2014) concluded that indeed ROS does not cause aging via indiscriminate damage and that they play a signaling role instead.</p>
<p>In 2015, Hou &amp; Amunugama notice inconsistent empirical results in the area, and try to reconcile them by proposing a model where ROS does cause damage after all. They argue that the antioxidant manipulations (Via gene modification and dietary supplementation), while effective in some cases in flies, failed to work on mice because without external stress, the rate of ROS generation is low enough not to be an issue. But subject to stress, not found in lab conditions, it would be. As for the studies that delete antioxidant-generating genes, they argue that other mechanisms for antioxidant reduction are upregulated, and while infant knockout mice present a higher level of nuclear DNA damage (due to ROS), their rate of increase is slower, so that those mice reach the same level of damage as their wild-type cousins at around the same time, so lifespan is unaffected.</p>
<p>In 2017, Rottenberg &amp; Hoek say that the evidence supporting the mFRTA is extensive, but that in recent years it has been suggested that while ROS may be generated in the mitochondria, they are fine with this, it is oxidative damage to nuclear DNA that is the problem. As for the inconsistent results, the authors note that knocking out SOD1, a gene coding for a key antioxidant protein, does reduce lifespan significantly, but that inducing acute moderate oxidative damage is no problem for the cell, as it can swiftly fix this. Pomatto &amp; Davies (2017) argue the same citing in vitro experiments with oxidant H2O2: A single exposure elicits a compensatory response, but chronic exposure produces lasting damage.</p>
<p>Ren &amp; Zhang (2018) note the existence of the “ROS and aging paradoxes” where there are evidence and theoretical arguments supporting and undermining a role for ROS in aging. They ascribe this to poor measurement and compensatory mechanisms (the antioxidant pathways being more redundant than it is thought).</p>
<p>Barja (2019) pushes back against the role of ROS as signaling molecules and again reasserts that the weight of the evidence is on the side of the oxidative theory.</p>
<p>Finally in a 2019 review, Munro &amp; Pamenter admit as of today no consensus has been reached, and they call for better measurement techniques. The authors even question the idea that mitochondria are even net sources of ROS (!). Where they say that there is consensus is that ROS damage to proteins floating around in the cytosol is not a driver of aging, but damage to the mitochondria may be.</p>
<p><img alt="" src="https://www.sciencefacts.net/wp-content/uploads/2020/04/Mitochondria.jpg" /></p>
<p>It is almost a meme to say that mitochondria are the powerhouses of the cell, but that’s what they really are. They are organelles that float around inside the cell enclosed in a double membrane that houses their own DNA (mitochondrial DNA, mtDNA); them being the only organelles that have this DNA. The regular nuclear DNA has genes that code for the ~1000 proteins that mitochondria need, except for 13 of them or so - the why might have to do with the fact that if the nucleus had those genes, then the proteins would have to be delivered into the mitochondria for the outside and apparently those 13 proteins are not easy to deliver there so they are made in-situ in mitochondrial ribosomes. Mitochondria are constantly being created (By fission of parent mitochondria) and destroyed (If they are damaged, lysosomes break them down); and mitochondria can also merge to form larger units</p>
<p>How do mitochondria generate energy? At a sufficiently high level, by creating an proton gradient between those two membranes. A set of molecular machines, complexes I to V pump the protons into it, which are then subsequently discharged through a rotating molecular turbine, ATP synthase that converts ADP into ATP (A useful molecule that is used for many other processes).</p>
<p>Cells have other ways of generating energy; they are generally fuelled by glucose, which is then broken down to pyruvate, releasing ATP in the process, then that pyruvate goes into the mitochondria where the complexes I-V take the hydrogen atoms (protons) in pyruvate and pump them into the double membrane. During these processes, the mitochondria are using another molecule as helper called nicotinamide adenine dinucleotide (NAD) which exists in two forms: NAD+ and NADH (The latter is the former plus an extra proton).</p>
<p>There have been theories proposed to explain part of the aging process resorting to ROS and mitochondria as noted in the earlier section, but it is worth mentioning the theory proposed by Aubrey de Grey - of SENS fame - which is not referenced in the reviews I have linked earlier but that seems to explain a bunch of the inconsistencies the research founds. In particular, the facts de Grey (2005) and (2002) starts from are:</p>
<p>Mitochondria generate ROS
Not much ROS does escape mitochondria (And thus they don’t affect proteins in the cell)
Because of the above, an increase of antioxidants in the cell does nothing to affect ROS or reduce damage
ROS damages mtDNA (And this is why evolution has been moving almost every gene from the mitochondria to the cellular nucleus, save for the 13 genes that are hard to migrate)
Deficiency in SOD2 (An antioxidant enzyme that works inside mitochondria) reduces lifespan in mice
Cells either contain healthy mitochondria, or (In a tiny minority of cells), mitochondria that all have the same mutation (in genes that code for some of the essential proteins for complexes I-V or ATP synthase) and are inactive.
His reasoning is then that while most damage to mtDNA leads to damaged mitochondria that are garbage-collected and replaced by new ones; if a mutation stops ATP production and thus ROS generation, the mitochondria won’t experience damage and will stick around; eventually, as other healthy mitochondria are replaced due to damage, given than the defective ones are never garbage-collected (As they are not damaged), those end up dominating the entire cell. Once this happens, this disrupts the balance of NAD+ to NADH which then the cell tries to balance out in a process that ends up exporting ROS outside the cell; this ROS then reacts with -among others- LDL chosterol, oxidising it, and then this damaged form of cholesterol can cause damage to cells elsewhere.</p>
<p>If damage to mtDNA is a cause of aging, a way to avoid it is to introduce copies of the remaining 13 genes in the celular nucleus. That way, regardless of what happens to mtDNA, there is always a way to make the relevant proteins. This is known as allotopic expression, and there has been success so far in moving 4 of those 13 genes in vitro (Made Artika, 2019)</p>
<p>Recently, it has been found that mitochondria can also be found circulating in the blood. It’s still unknown how they get there or if they play any function; likewise a recent finding is that cells can share mitochondria with each other. This is all to say that biology is incredibly complex and we are still discovering basic facts about it.</p>
</section>
<section id="cellular-senescence">
<h2>Cellular senescence<a class="headerlink" href="#cellular-senescence" title="Permalink to this headline">¶</a></h2>
<p><img alt="" src="https://www.mdpi.com/cells/cells-11-00672/article_deploy/html/images/cells-11-00672-g001.png" /></p>
<p>I’ve mentioned senescence earlier a couple of times. If a cell is damaged beyond repair, it can either die (apoptosis) and be recycled, or stick around and become a senescent cell. Senescent cells have a Senescence-Associated Secretory Phenotype (SASP) that in acute doses signals to the immune system that those cells should be cleared. But if there are too many senescent cells, then the SASP has truly negative consequences for the cells around them: A component of the SASP is the emission of cytokines and chemokines (proteins with signalling functions that regulate the activity of the immune system) to the extracellular medium that cause inflammation in neighbouring cells.</p>
<p>In aged individuals, the % of senescent cells is usually &lt;20% of the total number (Yang &amp; Sen, 2018) , although this varies by tissue (Tuttle et al. 2019). In mice, young individuals have 8% of senescent cells, compared to 17% in old age (Lopez-Otin et al. 2018)</p>
<p>To complicate things, senescent cells are not bad in general, they also help during embryonic development, tissue repair and wound healing, and of course the mechanism that turns cells senescent to begin with is in place to prevent cancer (Victorelli &amp; Passos, 2017). On that last point, it also has to be said that senescent cells, through inflammation of nearby cells, promote cancer (Lee &amp; Schmitt, 2019, Prieto &amp; Baker, 2019); and eventually a senescent cell can escape that state of arrest and turn cancerous too.</p>
<p>A commonality in senescent cells is the expression of the protein p16INKa which helps to identify them in vitro; however externally senescent cells are hard to distinguish from regular cells. A very recent review of it, a “SASP atlas” is Basisty et al. (2020), shows that there may even be different kinds of senescent cells, some of which have the positive effect noted earlier, and others just abnormal, negative effects.</p>
<p>Their effects are so negative that even transplanting a small amount of senescent cells to other mice causes them to live less and become less healthy (Xu et al., 2018)</p>
</section>
<section id="autophagy">
<h2>Autophagy<a class="headerlink" href="#autophagy" title="Permalink to this headline">¶</a></h2>
<p><img alt="" src="https://images.theconversation.com/files/436311/original/file-20211208-188518-1fyp5pw.png?ixlib=rb-1.1.0&amp;q=45&amp;auto=format&amp;w=754&amp;h=448&amp;fit=crop&amp;dpr=1" /></p>
<p>A mechanism cells have to fix damaged component is simply to break them apart, and produce new ones from scratch: autophagy (Bareja et al., 2019). It is at least intuitive to think that throwing something and building it again from scratch is easier than trying to repair a broken component, so it’s not that surprising that autophagy is involved in the alleviation of most if not all of the hallmarks of aging - to be discussed in the next section-.</p>
<p>Interventions that target the various longevity pathways (AMPK, mTOR, IIS, sirtuins) also tend to upregulate autophagy. Here’s a list of drugs that promote autophagy (From Shetty et al., 2018)</p>
<p><img alt="" src="https://nintil.com/images/2020-01-05-longevity/image-20200116074340112.png" /></p>
</section>
<section id="inflammation">
<h2>Inflammation<a class="headerlink" href="#inflammation" title="Permalink to this headline">¶</a></h2>
<p><img alt="" src="https://www.mypathologyreport.ca/wp-content/uploads/2021/12/inflammatory-cells.jpg" /></p>
<p>When the body detects pathogens, or otherwise anything that shouldn’t be there, specialised cells released small peptides called cytokines which attract immune cells (macrophages, lymphocites, etc) to the site to deal with the pathogen. There are various signs associated with it including pain, a burning sensation, redness and swelling; the root cause being vasodilatation to facilitate the arrival of immune cells to the area.</p>
<p>This all sounds good and reasonable, but what does it have to do with aging?</p>
<p>Remember the senescent cells? Part of the SASP is the emission of cytokines, a way of telling the immune system to clear them out. But if they are not (Because there are too many, or because the immune system is weakened) then the result is chronic inflammation around the whole body. Other factors also lead to this age-related kind of inflammation, which is typically referred to as inflammaging in the literature (Franceschi et al., 2018, Xia et al. (2016)). A related concept, metaflammation refers to inflammation for metabolic reasons due to overeating, a reason that helps explain why calorie restriction works: eating triggers an inflammatory response to prepare the body against potential pathogens in the newly ingested nutrients.</p>
<p>Inflammation is a response to a perceived threat that helps fight it, but inflammation even in the absence of any pathogen is perjudicial for the body; else we would have evolved to enjoy inflammation all the time. Inflammation swells tissues, which cause nearby tissues to expand and tear apart at the cellular level; the constant tearing up and rebuilding can lead to fibrosis (Mack, 2018). Also, the environment that inflammation creates is one that is meant to increase cell turnover (More apoptosis, but also more cell growth to replace lost cells), with granulocytes secreting toxic agents (Including ROS) to make the area affected less hospitable (But also increases damage to DNA), and specific cytokines like the tumor necrosis factor that induce cell death, and growth factors that promote cell growth. This increased turnover can facilitate the appearence of cancer.</p>
</section>
</section>
<section class="tex2jax_ignore mathjax_ignore" id="quantifying-biological-research-in-longevity">
<h1>Quantifying biological research in longevity<a class="headerlink" href="#quantifying-biological-research-in-longevity" title="Permalink to this headline">¶</a></h1>
<p>Once we have a very basic understanding of basic unit of life, the cell, the interesting question becomes:</p>
<blockquote>
<div><p><em>What can we measure about different processes happening inside of the cell?</em></p>
</div></blockquote>
<p>After all, if we can measure it, we can computationally tackle it.</p>
<p><a href="https://imgbb.com/"><img src="https://i.ibb.co/ryJRrpf/cell-measured.jpg" alt="cell-measured" border="0"></a></p>
<p>In molecular biology, as previously stated, the flow of genetic information is classically described by the <strong>Central Dogma</strong>, stating that genetic code is transcribed into RNA, which is in turn translated into proteins. These are the functional units which maintain cellular homeostasis and the source of cell type specific functions. Recently, developments in the fields of epigenetics and metabolomics have significantly challenged the Central Dogma. The genetic architecture was shown to directly affect genetic activity while the metabolism, previously thought of as one of the final outputs of the genome, was found to be a major source of input and genetic regulation by acting as an environmental sensor.</p>
<p>In Chapter 3, when talking about life-extension interventions we will reference the theory presented underneath and show how genomics, epigenomics etc. can be utilized for intervention purposes.</p>
<section id="genomics-data-for-longevity">
<h2>Genomics data for longevity<a class="headerlink" href="#genomics-data-for-longevity" title="Permalink to this headline">¶</a></h2>
<p><img alt="" src="https://cdn.medicalfuturist.com/wp-content/uploads/2018/10/Genomic-Big-Data-min.jpg" /></p>
<p>A different comparative approach to the genetics of ageing is to examine the association
of genetic variants, located across the entire genome of a species, with a phenotype.
These so-called genome-wide association studies (GWAS) have successfully identified
novel genes and pathway involved in various different complex phenotypes in humans.
In these studies, genetic variants across the entire genome are determined for individuals from two groups: people with a specific phenotype, e.g. a disease or old age, and
similar people without the phenotype. Frequencies of the observed variants are compared between the groups and statistically tested for differences, which would imply
association. A curated collection of all published GWAS that assay at least 100,000
single nucleotide polymorphisms (SNPs) and all SNP-trait associations with p-values
&lt; 1 ∗ 10−5
(genome-wide suggestive threshold) are published in the GWAS catalogue
(Hindorff et al., 2009, <a class="reference external" href="https://www.genome.gov/gwastudies/">https://www.genome.gov/gwastudies/</a>) (see Fig. 1.4 for the
map on longevity).</p>
<p>GWAS have been conducted for the phenotype of longevity, which is closely related to
ageing. Based on twin and population studies, the genetic contribution to longevity in
humans has been estimated to be 15 to 30%. The genetic contribution to survival past
65 and 85 years is even larger: 36% and 40%, respectively (Murabito et al., 2012). However, these GWAS on longevity have only had limited success so far. The first GWAS
study on longevity was the Framingham Heart Study 100K project (Lunetta et al.,
2007), which found no SNPs reaching genome-wide significance (p-value &lt; 5 ∗ 10−8
),
albeit a few with modest association.</p>
<p>Many subsequent studies and meta-analyses did
not find genome-wide significant results either, only suggestive associations (Malovini
et al., 2011; Newman et al., 2010; Walter et al., 2011) including SNPs in or near the
genes CAMKIV, OTOL1 and MINPPI. However, two studies, published around the
same time, identified significant genome-wide associations with longevity in humans.
Nebel et al. (2011) examined long-lived individuals and younger controls from Germany and found a single significant genome-wide association of SNP rs4420638, near
the APOC1 gene (p=1.8 ∗ 10−10).</p>
<p>However, this association is fully explicable by linkage disequilibrium with the 4 allele of the apolipo-protein E (APOE) gene, which had
been previously established in a non-genome wide study by Sch¨achter et al. (1994) to
be detrimental for longevity. The association of ApoE 4 with shortened longevity has
been replicated in several populations (Christensen et al., 2006). Additionally, ApoE
4 is more strongly associated with Alzheimer’s disease than with longevity (Murabito
et al., 2012). Overall, however, Beekman et al. (2010) showed that familial as well
as non-familial long-lived individuals “carry the same number of disease risk alleles
as young controls”, implying that the presence of disease risk alleles in general is not
linked to reduced longevity.</p>
<p>Deelen et al. (2011) examined nonagenarians (people 90
years of age or older) in comparison to young population-based controls and identified
the SNP rs2075650, located in TOMM40 on chromosome 19 as the sole genome-wide
significant association (p=3.39 ∗ 10−17). They found this SNP to be in moderate linkage with the nearby ApoE 4 (r
2 = 0.553), and could not find any APOE-independent
effect of rs2075650 on longevity. Thus they concluded that the SNP’s effect is mediated by APOE. Deelen et al. (2013) took this analysis a step further in using not
a GWAS of single SNPs, but of sets of SNPs grouped by pathway, which is similar
to the idea of comparing functionally enriched categories between different transcriptomics experiments. Several methods for using sets of SNPs in GWAS analysis had
been previously developed.</p>
<p>Deelen et al. (2013) tested the association of two SNP sets
with longevity: one SNP set for the IIS pathway and the other for the telomere maintenance pathway. Four different tests consistently showed a significant association of
both pathways with human longevity. Genome-wide association studies have also been
combined with genome-wide linkage analyses of long-lived siblings (Beekman et al.,
2013). The authors identified four regions which show linkage with longevity: 14q11.2,
17q11-q22, 19p13.3-p13.11 and 19q13.11-q13.32. They then created a model combining this information with a GWAS on a separate population. The combined model
demonstrated that the associations of lifespan with the ApoE 4 and ApoE 2 alleles
explain the linkage at 19q13.11-q13.32, while the other three linkage regions could not
be explained by variants detected in the GWAS for longevity, but might be interesting
regions to investigate further.
An alternative approach in human longevity association studies is the targeted examination of association, e.g. of genes previously associated with ageing in model organisms. This approach reduces the number of associations to be tested, thus making it
easier to detect significant associations. An example is the study by Bergman et al.
(2007) which examined 66 common variants in 36 known genes associated with cardiovascular disease in different age groups of long-lived Ashkenazi Jews and their offspring.</p>
<p>The authors examined the hypothesis that gene variants associated with longevity
might buffer against deleterious effects of age-related disease genes and identified three
candidate genes supporting the hypothesis: APOC-3, LPA, and CETP. The authors
then constructed a PPI network of these three genes and their third degree neighbours,
and analysed the minimal path connections between the initial genes. This implied
a few additional genes including APOE, LPAL2, PLTP and APOA1. Another target
study by Willcox et al. (2008) examined IIS genes and found Foxo3A to be associated
with human longevity in a long-lived male Japanese cohort. This association is the
only other highly replicated association with human lifespan (Anselmi et al., 2009;
Flachsbart et al., 2009; Li et al., 2009; Soerensen et al., 2010), besides ApoE 4. Other
associations such as the one of Paraoxonase 1 (PON1 ) are also the subject of targeted
association studies and even meta-analyses (Lescai et al., 2009), which suggest that
subjects carrying specific variants (RR or QR) of PON1 are over-represented in old
ages</p>
</section>
<section id="epigenomics-data-for-longevity">
<h2>Epigenomics data for longevity<a class="headerlink" href="#epigenomics-data-for-longevity" title="Permalink to this headline">¶</a></h2>
<p><img alt="" src="https://www.genome.gov/sites/default/files/inline-images/Epigenomics_Fact-sheet2020.jpg" /></p>
<p>Epigenetic changes are vital to normal biological functioning and can affect natural cycles of cellular death, renewal, and senescence. Different lifestyle and behavioral factors such as diet, sleep, exercise, smoking, and drinking alcohol can also affect the composition and location of the chemical groups that bind to our DNA. Environmental factors such as stress and trauma or even neighborhoods or zip codes may also have an impact.</p>
<p>One of the more prominent examples of  epigenomics data is DNA methylation. DNA methylation is the most researched epigenetic mark involved in various processes in the human cell, including gene regulation and development of diseases, such as cancer. Increasing numbers of DNA methylation sequencing datasets from human genome are produced using various platforms—from methylated DNA precipitation to the whole genome bisulfite sequencing. Many of those datasets are fully accessible for repeated analyses.</p>
<p>Consult <a class="reference external" href="http://gero.usc.edu/labs/benayounlab/files/2018/06/Ucar-18-Chapter.pdf">this</a> free online book a comprehensive overview.</p>
</section>
<section id="transcriptomics-data-for-longevity">
<h2>Transcriptomics data for longevity<a class="headerlink" href="#transcriptomics-data-for-longevity" title="Permalink to this headline">¶</a></h2>
<p><img alt="" src="https://cdn.kastatic.org/ka-perseus-images/20ce29384b2e7ff0cdea72acaa5b1dbd7287ab00.png" /></p>
<p>Large scale analysis of gene expression has been used extensively in research on ageing
(Girardot et al., 2006; McElwee et al., 2007; Pletcher et al., 2002; Selman et al., 2009;
Zahn et al., 2007). The wide availability and decreasing cost of microarrays have supported this development. More recently, RNA sequencing, with its increased sensitivity,
has gained much support in large scale gene expression determination and is expected
to supersede microarrays in the future. Currently, however, the significantly higher
costs and less well established protocols both experimentally and for the analysis are
limiting the use of RNA sequencing. In microarray experiments, the goal is to establish
differences in gene expression between two or more conditions: e.g. normal and disease,
different ages, or long lived and normal lived cohorts. RNA is prepared, hybridised to
the arrays, and raw binding data is read out via fluorescence imaging. The images are
then converted into numerical read outs, followed by bioinformatic analysis (detailed
in Gentleman et al. (2005, chapters 2, 4, 12-15)). Briefly, the raw numerical read out is
background corrected and normalised within the array and among arrays to eliminate
unspecific binding, differences in array production and other undesired factors, such
as different fluorescence properties of the dye. Then, the differentially expressed genes
and their statistical significance are determined, e.g. by fitting a linear model followed
by an empirical Bayes analysis. This approach is implemented in the R package limma
(Smyth, 2005).</p>
<p>An example demonstrating the use of gene-expression analysis for research on ageing
is the comparison of two well-known long-lived mutant mice strains (Ames dwarf mice
Prop1df /df and Little mice Ghrhrlit/lit) with the WT or heterozygote state, respectively
(Amador-Noguez et al., 2004). Using an analysis of variance (ANOVA) with p-value
≤ 0.0001 the authors found 1125 and 1152 differentially expressed genes between mutant and control for each strain, respectively. 552 of these genes overlapped between
the two mutants. In another study, Landis et al. (2012) used microarrays to determine
the change in gene expression during ageing, but also in response to various types of
stress including heat stress, oxidative stress and ionizing radiation. The authors then
examined the overlap of the changes to determine common characteristics and found
18 genes to be up-regulated across all conditions. In a different approach, Spindler &amp;
Mote (2007) used microarrays profiles to compare long-term and short-term DR to the
effect of glucoregulatory drugs. They found that metformin produced a similar expression profile compared to DR in mice and suggested that expression profiles could be
used as an easier way of screening drugs compared to lifespan studies.</p>
<p>Lists of differentially expressed genes can be easily used for determining the similarity
between two or more conditions, however, interpreting changes in hundreds or thousands of genes is very difficult. In order to summarise these into broader patterns two
approaches are commonly used: functional enrichment analysis and motif analysis.
Functional enrichments analysis searches for over-represented gene annotations such as
GO terms or disease-association amongst the differentially expressed genes compared
to the background. It is readily available through tools such as Catmap (Breslin et al.,
2004) and online services such as DAVID (Huang et al., 2009), and has been applied
in various cases including comparative studies (see section 1.3.7). The mouse study
mentioned above (Amador-Noguez et al., 2004) found an over-representation of genes
involved in metabolism and mitochondrial electron transport. While these broader
categories might provide useful overviews, new tools are needed to prioritise genes for
follow-up studies. Such genes should represent maximally different functional categories while having the largest impact on the phenotype. Such a prioritisation will
need to take into account protein-protein and gene-regulatory network relationships.
In motif analysis, the grouping is not based on functional annotations, but on joint
transcriptional regulation by the same transcription factor indicated by shared binding motifs. A list of differentially expressed genes can either be searched for over- or
under-represented motifs from databases of known motifs, or attempts can be made
to identify motifs ab initio. The advantages of methods using databases, also called
library-based methods, such as Clover (Frith et al., 2004), are sensitivity in detecting
weak motifs and inclusion of information on which transcription factor corresponds
to which motif, if this is known. There are open-source and commercial databases of
motifs, e.g. JASPAR (Portales-Casamar et al., 2010) or TRANSFAC (Matys et al.,
2003). Methods trying to detect motifs ab initio, such as NestedMICA (Dogruel et al.,
2008), are less sensitive, but have the potential to discover novel motifs. An example
of a motif analysis is the study of three long-lived mutants (sch9 ∆, ras2 ∆, and tor1 ∆)
and wild type yeast (Cheng et al., 2007). In that study, the authors found a number of
enriched motifs including those that correspond to the transcription factors Fhl1 and
Gis1. These transcription factors regulate expression of ribosomal protein genes and
stress-response genes, respectively.</p>
<p>A different approach to detect co-regulation was used by Southworth et al. (2009), who
looked for correlation between patterns of differential gene expression across different
tissues of mice. They constructed a co-expression network, which defines modules of
co-expressed genes. These modules were then characterised using GO categories and
motif analysis. Overall, the authors found that with age, the co-expression declined
within the modules.</p>
<p>Transcriptomics microarray analysis was used in a very different way for research on
ageing by Matecic et al. (2010). Their study screened a large library of yeast mutants
containing DNA bar codes for survival differences. To this end, all viable mutants
were pooled and cultured under high and low glucose levels. The mixed population
was sampled on different days (1, 9 ,21 and 33) and microarrays were used to identify differential expression of the bar codes among the different times, indicating the
chronological lifespan of the respective strains.
Besides the problems of interpreting a large number of genes with altered genes expression and prioritising follow up experiments, discussed above, one problem of transcriptomics is that often not the changes in RNA levels are of interest but the changes
in protein concentration or even signalling activity, which do not correlate in all cases.</p>
</section>
<section id="proteomics-data-for-longevity">
<h2>Proteomics data for longevity<a class="headerlink" href="#proteomics-data-for-longevity" title="Permalink to this headline">¶</a></h2>
<p><img alt="" src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTZiGDKTDHA--vyio7qTW6TrjJ_b6SLkLBEDQ&amp;usqp=CAU" /></p>
<p>In contrast to transcriptomics, which examines RNA expression levels using microarrays or next-generation sequencing, proteomics and metabolomics are the profiling of
protein and metabolite levels, respectively, usually using advanced mass spectrometry.
In proteomics, the focus can be on different aspects from relative protein concentrations
to protein modifications and complex formation. Proteomics studies for questions of
research on ageing are relatively rare, certainly when compared to gene expression
analyses. However, the number of such studies is increasing (reviewed by Schiffer et al.,
2009; Sharov &amp; Schoneich, 2007). An example is the analysis of changes in protein
concentration in the long-lived daf-2 mutants in C. elegans (Jones et al., 2010).</p>
<p>The
authors found changes for proteins related to cellular maintenance and detoxification
processes, in agreement with previous transcriptional analyses. A recent study by
Depuydt et al. (2013) also analysed the protein changes in daf-2 mutant in C. elegans
and found general protein synthesis levels and in particular the abundance of ribosomal
subunits decreased. Surprisingly, their analyses found that ribosomal transcript levels
were not correlated to actual protein abundance, suggesting that post-transcriptional
regulation would determine ribosome content and highlighting the uncertainties in
general agreement between transcriptomics and proteomics analyses. A different example of a proteomics study is the study by Robinson et al. (2010), which examined
changes in the proteome of Drosophila males reared at two different temperatures
over their lifespan. Using liquid chromatography and tandem mass spectrometry the
authors found 33 proteins that shared similar patterns when using scaling time points
according to the group’s lifespan, instead of the absolute time points.</p>
<p>For the most recent advances of proteomics and involvement in Aging look <a class="reference external" href="https://pubmed.ncbi.nlm.nih.gov/33730416/">here</a>.</p>
</section>
<section id="metabolomics-data-for-longevity">
<h2>Metabolomics data for longevity<a class="headerlink" href="#metabolomics-data-for-longevity" title="Permalink to this headline">¶</a></h2>
<p><img alt="" src="https://www.mtidx.com/public/uploads/images/mti_metabolomics/1578509820-640w_mti_metabolomics.jpg" /></p>
<p>Metabolites such as glucose and oxygen species have been shown to be important in
ageing, and some metabolomics studies on ageing have been conducted. An example
is the analysis of membrane phospholipid composition in long-lived naked mole rats
compared to short-lived mice (Mitchell et al., 2007). The statistical analysis of mass
spectrometry measurements indicated marked differences between the two species,
e.g. lower level of docosahexaenoic acid containing phospholipids, which suggests a
lower susceptibility to peroxidative damage in membranes of naked mole-rats compared
to mice. Yoshida et al. (2010) used metabolic profiling of various yeast mutants and
discovered a correlation between the metabolic profiles and yeast replicative lifespan
(i.e. how many rounds of replication a yeast cell will undergo). They further developed
a multi-variate model based on the metabolic profiles to predict yeast replicative lifespan. This model allows screening for lifespan mutants in yeast using metabolic profiles
as the read out. Using nuclear magnetic resonance, Fuchs et al. (2010) examined the
metabolome of various longevity mutants of C. elegans, including daf-2, and found a
common metabolic signature, which might indicate that there are hidden connections
among diverse longevity mechanisms.</p>
<p>For the most recent advances of Metabolomics Signatures of Aging look <a class="reference external" href="https://pubmed.ncbi.nlm.nih.gov/33815888/">here</a>.</p>
</section>
<section id="multi-omics-approach-in-longevity">
<h2>Multi -omics approach in longevity<a class="headerlink" href="#multi-omics-approach-in-longevity" title="Permalink to this headline">¶</a></h2>
<p><img alt="" src="https://www.researchgate.net/profile/Ashok-Kumar-T-2/publication/320324303/figure/fig1/AS:549025272090629&#64;1507909233497/An-illustration-of-multi-omics-data-integration-through-integrative-systems-biology.png" /></p>
<p>Omics analysis is usually used in conjunction with other assays and I will briefly highlight a few examples of multiple -omics approaches. One example is the combination
of gene expression analysis with information on genome variations in the form of single
nucleotide polymorphisms (SNPs). Wheeler et al. (2009) analysed gene expression in
human kidney cells and found 630 genes with age-dependent changes in gene expression,
101 of which contained expression-associated SNPs. The two staged approach used here
reduced search space and thus the number of statistical tests for association, making it
easier to detect significant effects. A different example combined gene expression transcriptomics with metabonomics (Selman et al., 2006). In this study, gene expression
from liver, skeletal muscle, colon and hypothalamus of acute dietary restricted mice
were determined. The expression profiles together with blood plasma metabonomic
profiles lead the authors to conclude that “mice undergoing acute CR rapidly adopt
many transcriptional and metabolic changes of long-term CR” (Selman et al., 2006).</p>
<p>A potentially interesting study on ageing could be the combination of different -omics
data sets available on several long-lived mutants such as daf-2 in C. elegans (microar-
rays (Murphy et al., 2003), proteomics (Dong et al., 2007) and metabolomics (Fuchs
et al., 2010)).
Recently, the genome of the naked mole rat, the longest living rodent with a maximal
lifespan larger than 30 years, has been sequenced using next-generation sequencing
(Kim et al., 2011). This animal has not only a remarkably long lifespan, but is also
reported to have other extraordinary characteristics including negligible senescence, no
age-related increase in mortality, high fecundity until death as well as resistance to
spontaneous and induced cancers. The study by Kim et al. (2011) not only sequenced
and assembled the genome, but also predicted genes and gene functions, analysed
lost and gained genes and their functional categories as well as their association with
phenotypic characteristics. The authors additionally identified 1.87 million SNPs and
measured gene expression in brain, liver and kidney at three different ages (new-born,
young adult (4 years), old adult (20 years)) using RNA-seq. Differential gene expression analysis revealed few changes between young and old adults compared to other
mammals. Overall, genome and transcriptome data showed distinct patterns of naked
mole rat genes compared to humans, mice and rats, and these patterns might include
those genes important for the different lifespans. One example is the lower expression
of IIS genes in the liver of naked mole rats compared to mice.</p>
<p>Comprehensive database, Aging Atlas, can be found <a class="reference external" href="https://academic.oup.com/nar/article/49/D1/D825/5943197">here</a>.</p>
</section>
<section id="network-biology-approach-in-longevity">
<h2>Network biology approach in longevity<a class="headerlink" href="#network-biology-approach-in-longevity" title="Permalink to this headline">¶</a></h2>
<p><img alt="" src="https://media.springernature.com/m312/springer-static/image/art%3A10.1038%2Fnrg1272/MediaObjects/41576_2004_Article_BFnrg1272_Fig2_HTML.jpg?as=webp" /></p>
<p>The premise is that with the explosion of data and the complex intrecacies of the relationships between all of the different (multi-omics) datasets, there needs to be a quantitative way to represent and find these networks of relationship.</p>
<p>Large-scale consortium-based efforts looking at the various aspects of human biology have allowed the application of network-based methods to uncover new insights into the molecular mechanisms of the given phenotype, such as tissue specificity or disease context.</p>
<p>Examples of real world biological networks:</p>
<ul class="simple">
<li><p><strong>Protein-Protein Interaction Networks (PPIs)</strong>
This type of networks holds information about how different proteins operate with each other to enable a biological process within a cell. The interactions in a PPI network can be physical or predicted. Notably, a whole interactome can capture all PPIs happening in a cell or an organism. In vivo and in vitro methods for detecting PPIs include: X-ray crystallography, NMR, tandem affinity purification (TAP), affinity chromatography, coimmunoprecipitation, protein arrays, protein fragment complementation, phage display and yeast two-hybrid (Y2H). Widely used repositories (Lehne and Schlitt, 2009; Szklarczyk and Jensen, 2015) which host PPIs for various organisms are the BioGRID (Stark et al., 2006), MINT (Chatr-aryamontri et al., 2007), BIND (Bader et al., 2003), DIP (Xenarios et al., 2000), IntAct (Hermjakob et al., 2004a), and HPRD (Peri et al., 2003) database. Concerning topology, the PPI networks follow a small-world property and are scale-free networks. Central hubs often represent evolutionarily conserved proteins, whereas cliques (fully connected subgraphs) have been found to have a high functional significance (Spirin and Mirny, 2003).</p></li>
<li><p><strong>Sequence Similarity Networks (SSNs)</strong>
These networks consist of nodes representing proteins or genes and edges capturing the sequence similarity between amino acid or nucleotide sequences. Widely used tools (Ekre and Mante, 2016) for obtaining a sequence similarity between two sequences are the BLAST (Altschul et al., 1990), LAST (Kiełbasa et al., 2011), and FASTA3 suite (Pearson, 2000), which contains SSEARCH, GGSEARCH, GLSEARCH executables of Smith-Waterman (Smith and Waterman, 1981) and Needleman-Wunsch (Needleman and Wunsch, 1970) implementations for local and global sequence alignment. These networks are weighted, have a small-world and scale-free topology and often contain hubs. Often, clustering algorithms are applied on such networks for the detection of protein families. Like in PPIs, proteins that lie together in such networks are more likely to have similar functions or be involved in similar biological processes (Sharan et al., 2007). While it is not straightforward to come to a conclusion about their density, when coping with fragmented sequences (e.g., alignments of predicted proteins from metagenomes), the networks are rather sparse.</p></li>
<li><p><strong>Gene Regulatory Networks</strong></p></li>
</ul>
<p><img alt="" src="https://upload.wikimedia.org/wikipedia/commons/c/c4/Gene_Regulatory_Network.jpg" />
A gene (or genetic) regulatory network (GRN) is a collection of molecular regulators that interact with each other and with other substances in the cell to govern the gene expression levels of mRNA and proteins which, in turn, determine the function of the cell.
They are collections of regulatory relationships between transcription factors (TFs) and TF-binding sites or between genes and their regulators. Normally, these networks are directed, dynamic, and can be visualized as bipartite graphs. In such networks, most nodes have only a few interactions and only a few hubs come with a higher connectivity degree. In any case, such networks follow a power law degree distribution (scale-free) p(k) ~ k−γ, γ ≈ 2 (Vázquez et al., 2004). Among a variety of databases hosting information about gene regulation, widely used repositories are the KEGG (Kanehisa and Goto, 2000), GTRD (Yevshin et al., 2019), TRANSFAC (Matys et al., 2003), TRRUST (Han et al., 2018).</p>
<ul class="simple">
<li><p><strong>Signal Transduction Networks</strong>
These networks capture cell signaling or otherwise the transmission of molecular signals as well as a series of molecular events within a cell or from the exterior to its interior (Fabregat et al., 2018). A signal transduction network normally consists of several thousand nodes and edges representing a series of reactions. These networks are mostly directed and sparse. They follow a power law degree distribution as well as small-world properties. While such data can be found in well-known pathway databases (KEGG, Reactome), specialized repositories such as the MiST (signal transduction in microbes) (Ulrich and Zhulin, 2007), NetPath (Kandasamy et al., 2010), or Human-gpDB (Satagopam et al., 2010) also exist.</p></li>
<li><p><strong>Metabolic Networks</strong>
They are networks consisting of metabolites (nodes) and their interactions in an organism. Metabolites can be either smaller molecules such as amino acids or larger macromolecules like polysaccharides. These networks are usually directed graphs and can be represented as Petri nets (Reisig, 1985; Chaouiya, 2007). They are scale-free, they carry small-world properties (Jeong et al., 2000) and can often be organized using hierarchies (Gagneur et al., 2003). In order to gain insights into their decomposition, heuristic modularity optimization over all possible divisions to find the best one is required (Newman and Girvan, 2004). KEGG and Reactome databases are two of the most widely used repositories for this type of network.</p></li>
<li><p><strong>Gene Co-expression Networks</strong>
They are undirected weighted networks where two nodes (genes) are connected if there is a significant co-expression between them. Such networks are usually constructed using data from high-throughput technologies such as Microarrays, RNA-Seq or scRNA-seq. For each pairwise connection, a metric like for example, the Pearson Correlation Coefficient (PCC) (Kirch, 2008) can be used to calculate an edge’s weight. Often, a threshold or a Z-score are applied on the whole network in order to accept correlations above a certain cutoff. Otherwise the network would look like a fully connected clique. After the threshold and depending on the total clustering coefficient, the network can be clustered to detect functional modules. One typical example is the ribosomal genes which tend to group together due to similar expression patterns. Expression data for such analyses can be found in widely used repositories such as GEO (Barrett et al., 2013) or ArrayExpress (Parkinson et al., 2007). Notably, Arena-Idb (Bonnici et al., 2018) repository can be used for human non-coding RNAs interactions.</p></li>
</ul>
<p>Potential applications of network biology:</p>
<ul class="simple">
<li><p>Clustering — finding families and centralities inside big graphs</p></li>
<li><p>Network Alignment — In today’s multi-Omics era, integration of heterogeneous information (e.g., transcriptomics, proteomics, metabolomics, etc.) in a multi-layered network structure is becoming a trend. Additionally, methods to directly compare networks and their topological features are gaining ground. To address these issues, network alignment, or alternatively graph isomorphism approaches can be used. Notably, graph alignment is not a trivial task as it is computationally expensive and has been characterized as NP-complete (Zampelli et al., 2010). The concept behind network alignment is to highlight conserved or missing nodes and edges across two (pairwise) or more (multiple) networks.</p></li>
<li><p>Link prediction — Besides network alignment, predicting link changes in a single network has recently drawn attention in the biomedical field. Link prediction might concern the creation of future edges or the identification of missing links (e.g., incomplete data). While link prediction techniques are widely used by social media, in biological networks, they have also been used to identify potential drug side effects, protein-protein interactions, disease phenotypes based on molecular information and phylogenetic relations. For example, its application on bipartite graphs has unraveled new drug-target interactions (Kunegis et al., 2013).</p></li>
<li><p>Network Perturbation — In biology, direct comparisons between a disease and a healthy state are very common, thus making the study of molecular changes essential. Therefore, at a network level, changes between such states are considered as biological network perturbations. In network medicine, a network’s topology can be used as the backbone to further predict side effects in a system even at a 65–80% success rate (Santolini and Barabási, 2018).</p></li>
</ul>
<p>How to construct and the perform graph topology analysis in order to extract insight:</p>
<p><img alt="" src="https://i.postimg.cc/L6nvxfmP/Screenshot-2022-04-20-075723.jpg" /></p>
</section>
<section id="general-measurament-tools">
<h2>General Measurament tools<a class="headerlink" href="#general-measurament-tools" title="Permalink to this headline">¶</a></h2>
<p>I am going to name a couple of prominent methods important for capturing information on different levels, from proteomics to genomics. Check <a class="reference external" href="https://mikeant.medium.com/debugging-biology-e0b4e809b791">here</a> for the source and a great read.</p>
<ul class="simple">
<li><p><strong>Sequencing</strong> — reads the structure of DNA and RNA.</p></li>
<li><p><strong>qPCR</strong> — Real Time polymerase chain reaction, which monitors amplification of DNA with help of fluorescence. Useful for diagnostics, quantification of gene expression.</p></li>
<li><p><strong>Antibody-based Methods</strong> — Western Blots, ELISAs, and multiplex assays to analyze the presence and quantity of proteins.</p></li>
<li><p><strong>X Ray-crystallography and Cryo-EM</strong> — For structural biology, identifying the structure of molecules and proteins, including detail such as conformation and location of atoms.</p></li>
</ul>
<section id="sequencing">
<h3>Sequencing<a class="headerlink" href="#sequencing" title="Permalink to this headline">¶</a></h3>
<p>DNA and RNA sequencing tools read the chain of nucleic acids, decoding them into sequences of 2-bit bases representing Cytosine, Guanine, Adenine, Thymine (or Uracil in RNA). These sequences are important because they contain genes that guide creation of proteins, their regulation and other machinery of life. A switch in one base could replace an amino acid in a protein, breaking its functionality and introducing disease; alternatively, it could also provide an evolutionary advantage. The whole human genome contains around 3.1 billion bases (or 6.2 billion if counting both chromosomes), and the major scientific achievement of the last twenty years was first determining the structure of these bases, and then dropping the cost of doing so to just a few hundred dollars today.</p>
<p>DNA sequencing has huge value in research as it allows us to study genes, including their differences between biological species, and with help of bioinformatics software attempt to map them to function and phenotypes. RNA sequencing allows us to understand transcription, or which genes are expressed in different cell types. Beyond this, sequencing is also used indirectly to study methylation, chromatin binding, or even perform protein assays with help of custom DNA adapters to antibodies. Overall, these tools help us better understand cellular state and function.</p>
<p>There are many different technical approaches to sequencing such as sequencing by synthesis done by Illumina, where a fluorescent signal is emitted and read every time a nucleotide is added; or detecting ionic current changes as DNA strand passes through a protein channel in a device membrane, as in Oxford Nanopore. Scanning a DNA strand this way generates a “read”, which depending on the technology provider is either short (~150–200 bases, Illumina) or long, such as 1500 bases+. Scientists pick the approach based on their goals, as well as cost and error rate:</p>
<ul class="simple">
<li><p>Short reads (Illumina, Ion Torrent, BGI Genomics) are often cheaper and have low error rates, often under 0.5% per base. They are great for detecting SNPs/variations which could cause a disease, but make it harder to build the whole genome without reference.</p></li>
<li><p>Long reads (Pac Bio, Oxford Nanopore) approaches have higher error rates and are generally better for de-novo construction of the genome. This can make them better for metagenomic analysis, which includes sequencing many organisms at once for understanding a given microbiome or studying the population of organisms in a lake.</p></li>
</ul>
<p>Ultimately, tens of thousands to millions of such read segments can be generated per run producing gigabytes of data; alignment software tools such as GSNAP or Bowtie 2 are then used to map it to the reference genome. Once reads are mapped, you can check whether there is a difference/mutation in the nucleotides of a given gene and perform other analyses.</p>
</section>
<section id="qpcr">
<h3>qPCR<a class="headerlink" href="#qpcr" title="Permalink to this headline">¶</a></h3>
<p>qPCR — Quantitative Polymerase Chain Reaction is commonly used to quantify gene expression, i.e. the level of a given RNA in the sample. This might be useful in an experiment to compare gene expression after a certain gene knockout or pathway modification, or in diagnostics to detect the presence of a certain virus.
PCR utilizes the power of the polymerase enzyme to amplify a DNA sequence of interest. This method makes use of a thermocycler that repeatedly denatures DNA separating its strands, allowing the enzyme to copy a specific region defined by the primer sequences. As a result that specific sequence of interest is amplified over a billion times and can thus be detected. In case of qPCR, the RNA in the sample is first converted into DNA using a reverse transcriptase enzyme and then the regular PCR is run on the resulting so-called cDNA. Its abundance is detected using fluorescence, which corresponds to the level of expression and can be visualized with amplification curves.</p>
<p><img alt="" src="https://miro.medium.com/max/1400/1*BIMF1Se3WpdOg3_GKnWEzw.png" /></p>
<p>Gene expression results of qPCR might be reported in a scientific paper with an illustration similar to the one above, where RNA_1 expression was knocked down, potentially with help of a viral vector or short hairpin RNA, while a different RNA_2 is not affected.</p>
</section>
<section id="antibody-based-methods">
<h3>Antibody-based Methods<a class="headerlink" href="#antibody-based-methods" title="Permalink to this headline">¶</a></h3>
<p>Antibody-based detection methods are used extensively in science to detect and quantify proteins and other molecular structures, as well as to stain tissue to reveal analyte location. Protein quantification is important for both research and diagnostics: on research side, protein measurement aides with mechanistic understanding of the cell function; on the diagnostic side, it enables disease detection — as an example, levels of C-reactive protein (CRP) rise in response to tissue damage and therefore are used for detection of heart disease.
Protein quantification typically relies on protein-specific antibodies being conjugated with a unique DNA strand, a bead or an enzyme that aids detection. In case of an enzyme, it can be used to catalyze a reaction that triggers color change, observed to detect its presence.
There are hundreds of companies providing tools and reagents for antibody-based assays, so here I’ll highlight two classical methods still used in research — Western Blot and ELISA, and mention a few more advanced modern techniques:</p>
<p><img alt="" src="https://upload.wikimedia.org/wikipedia/commons/thumb/0/04/Anti-lipoic_acid_immunoblot.png/220px-Anti-lipoic_acid_immunoblot.png" /></p>
<ul class="simple">
<li><p>Western Blot is one of the cheapest and most common methods of protein detection; combining gel separation of proteins based on molecular weight/charge with antibody staining to make them visible as a band. It is semi-quantitative, as you can interpret the size of the band but it is hard to do precisely; when necessary, you can literally cut out a band out of the gel for further analysis, such as with mass spec. If you’ve seen images of vertical gel images in scientific papers, this is probably it (or similar northern/southern blot, used to detect RNA/DNA).</p></li>
<li><p>ELISA, or “Enzyme Linked Immunosorbent Assay” is a plate-based method used to detect and quantify the abundance of proteins/substances with the help of antibodies. ELISAs are designed in such a way that when the detection antibody binds to a protein of interest, it stays within the plate well; otherwise it is washed away. Enzymes bound to such remaining antibodies catalyze a color change in a fluorescent reagent, so the protein is detected.</p></li>
<li><p>ELISAs are often done using 96-well plates that you can get in pre-made kits for a given protein, such as Invitrogen IL-8 ELISA Kit. Several samples can be processed on the same plate, with multiple dilution levels each. Due to its good sensitivity and a broad dynamic range, ELISAs are seen as the gold standard for protein quantification.</p></li>
</ul>
<p><img alt="" src="https://miro.medium.com/max/1400/1*GojpdXYpUh4pU6pAPDzcoQ.png" /></p>
<p>There’s been great progress in Multiplexing methods over the last two decades, allowing one to quantify many, often hundreds of proteins in the same sample at once — useful for analyzing protein expression and smarter diagnostics. Two common approaches are bead-based “suspension arrays” or planar arrays.</p>
<ul class="simple">
<li><p>Bead-based approaches, as with Luminex, use microscopic antibody-coated beads (2.5M beads/mL), mixed with a sample. Such beads can be separated based on the unique wavelength of light with flow cytometry to quantify each type of protein.</p></li>
<li><p>Planar arrays (ex: RayBiotech and FullMoonBio) place antibodies in a predetermined pattern on a surface, used to capture molecules and display them as an expression map.</p></li>
<li><p>Alternative technologies, such as Olink® Explore 1536/384, make use of antibody pairing and DNA hybridization to convert protein binding “signal” into DNA that can be sequenced. You get good sensitivity and much higher multiplexing to handle many hundreds of samples per run, yet limited quantification capabilities due to reliance on sequencer reads.</p></li>
<li><p>Highest sensitivity approaches today, such as Quanterix Simoa, surpass ELISA and are able to detect and quantify proteins at femtogram level, which essentially requires single molecule capture. For now, these panels are still limited to 10-pexity per sample and involve expensive equipment.</p></li>
</ul>
<p>One of the biggest challenges with protein quantification is caused by the high dynamic range of protein concentrations, which can vary by 10^12, making it hard to detect rare low-concentration molecules crowded out by the higher-concentration ones. The other one is the cost and availability of specific antibodies to protein targets: while we have thousands of antibodies, they do not cover the entire human proteome. There are time/ease use considerations as well, as sometimes reagents or steps require hours of preparation and processing.</p>
<p>Nevertheless, this area of proteomics is advancing very fast. A lot of money is going into developing higher plexity and sensitivity methods, coupled with AI will help us diagnose a wide range of diseases early. Because of this I would expect more high-sensitivity multiplex, cow-cost methods to become available over the next few years.</p>
</section>
<section id="x-ray-crystallography-and-cryo-em">
<h3>X-Ray Crystallography and Cryo-EM<a class="headerlink" href="#x-ray-crystallography-and-cryo-em" title="Permalink to this headline">¶</a></h3>
<p>X-Ray Crystallography and Cryo-EM are methods for identifying the structure of molecules and proteins, including their conformation and individual atom coordinates. This is important since protein folding shape determines its function.</p>
<ul class="simple">
<li><p>X-Ray crystallography has been the primary method for solving protein structure with over 150,000 structures in Protein Data Bank, compared to 13,000 for NMR and 6000 for Cryo-EM. It requires formation of crystals with regular internal protein lattice. It then determines the atomic/molecular structure of the crystal by analyzing angles of diffracted x-ray beams, processed to compute chemical bonds and mean positions of atoms.</p></li>
<li><p>Cryo-EM reconstruction applies electron microscopy to samples cooled to cryogenic temperatures in vitreous water. Technology broke the 3 angstrom resolution barrier in 2015 with advances in direct-electron detectors, so the number of Cryo EM submissions to PDB has been accelerating in recent years.</p></li>
</ul>
<p><img alt="" src="https://miro.medium.com/max/1400/0*6hPZf3WHwTJlsFla" /></p>
<p>One of the benefits of Cry-EM is that it avoids crystallization, which can be very time consuming and difficult to prepare. It is better suited for studying membrane proteins which can be hard to separate for crystallization and larger molecules/protein assemblies typically bigger than 60 kDA in size, as low molecular mass molecules which have fewer scattering atoms contribute less to the signal. X-Ray crystallography tends to be more useful for smaller proteins, also resulting in higher resolution and precision in atomic coordinates, aided by the regular crystal structure. Here is an interesting article on X-rays in Cryo-EM Era.</p>
</section>
</section>
<section id="data-sources">
<h2>Data sources<a class="headerlink" href="#data-sources" title="Permalink to this headline">¶</a></h2>
<p>Now that we have named the tool and the terms that we want to measure, here are central repositories containing the afromentioned data.</p>
<section id="data-sources-specifically-dedicated-to-aging">
<h3>Data sources specifically dedicated to aging<a class="headerlink" href="#data-sources-specifically-dedicated-to-aging" title="Permalink to this headline">¶</a></h3>
<p>One of the central resources is <a class="reference external" href="https://academic.oup.com/nar/article/46/D1/D1083/4599180?login=true"><strong>HAGR</strong> (Human Ageing Genomic Resources)</a> containing data not only commited to the genomic aspects of aging.</p>
<ul class="simple">
<li><p><strong><a class="reference external" href="https://genomics.senescence.info/genes/index.html">GenAge</a></strong>—the ageing gene database.</p></li>
<li><p><strong><a class="reference external" href="https://genomics.senescence.info/diet/">GenDR</a></strong>—a database of dietary restriction-related genes.</p></li>
<li><p><strong><a class="reference external" href="https://genomics.senescence.info/longevity/">LongevityMap</a></strong>—human genetic variants associated with longevity.</p></li>
<li><p><strong><a class="reference external" href="https://genomics.senescence.info/cells/#:~:text=To%20facilitate%20studies%20focused%20on,cellular%20senescence%20is%20also%20available.">CellAge</a></strong>—a database of cell senescence genes.</p></li>
<li><p><strong><a class="reference external" href="https://genomics.senescence.info/species/index.html">AnAge</a></strong>—the database of animal ageing and longevity (Comparative biology).</p></li>
<li><p><strong><a class="reference external" href="https://genomics.senescence.info/drugs/">DrugAge</a></strong>—a database of ageing-related drugs (Chemistry of aging).</p></li>
<li><p><strong><a class="reference external" href="https://academic.oup.com/nar/article/49/D1/D825/5943197">Aging Atlas</a></strong>—a multi-omics database for aging biology.</p></li>
<li><p><strong><a class="reference external" href="https://jaspar.genereg.net/">Jaspar</a></strong>—a database of transcription factor binding profiles.</p></li>
<li><p><strong><a class="reference external" href="https://www.genome.jp/kegg/">KEGG</a></strong>—a database of molecular pathway maps and categories.</p></li>
<li><p><strong><a class="reference external" href="https://www.uniprot.org/">UniProt</a></strong>—a database of protein sequence and functional information</p></li>
<li><p><strong><a class="reference external" href="http://agefactdb.jenage.de/">AgeFactDB</a></strong>—a database integrating GenAge and Lifespan Observation DB..</p></li>
<li><p><strong><a class="reference external" href="http://cmgm.stanford.edu/~kimlab/aging_mouse/">AgeMap</a></strong>—a database integrating gene expression for aging in mice.</p></li>
<li><p><strong><a class="reference external" href="https://gisao.genome.tugraz.at/gisao_web/">GiSAO</a></strong>—a database  of genes involved in Senescence, Apoptosis and Oxidative Stress.</p></li>
<li><p><strong><a class="reference external" href="http://agingportfolio.org/">Aging Research Portfolio</a></strong>—a databse on aging research world wide.</p></li>
<li><p><strong><a class="reference external" href="https://www.uniprot.org/">UniProt</a></strong>—a databse of protein sequence and functional information</p></li>
<li><p><strong><a class="reference external" href="https://www.uniprot.org/">UniProt</a></strong>—a databse of protein sequence and functional information</p></li>
</ul>
</section>
</section>
</section>

    <script type="text/x-thebe-config">
    {
        requestKernel: true,
        binderOptions: {
            repo: "binder-examples/jupyter-stacks-datascience",
            ref: "master",
        },
        codeMirrorConfig: {
            theme: "abcdef",
            mode: "python"
        },
        kernelOptions: {
            kernelName: "python3",
            path: "./."
        },
        predefinedOutput: true
    }
    </script>
    <script>kernelName = 'python3'</script>

              </div>
              
            
                <!-- Previous / next buttons -->
<div class='prev-next-area'> 
    <a class='left-prev' id="prev-link" href="Chapter%201%20-%20Motivation.html" title="previous page">
        <i class="fas fa-angle-left"></i>
        <div class="prev-next-info">
            <p class="prev-next-subtitle">previous</p>
            <p class="prev-next-title">Chapter 1 - Motivation</p>
        </div>
    </a>
    <a class='right-next' id="next-link" href="Chapter%203%20-%20Hallmarks%20and%20biomarkers%20of%20aging.html" title="next page">
    <div class="prev-next-info">
        <p class="prev-next-subtitle">next</p>
        <p class="prev-next-title">Chapter 3 - Hallmarks and biomarkers of aging</p>
    </div>
    <i class="fas fa-angle-right"></i>
    </a>
</div>
            
        </div>
    </div>
    <footer class="footer">
  <p>
    
      By Noah Weber<br/>
    
        &copy; Copyright 2022.<br/>
  </p>
</footer>
</main>


      </div>
    </div>
  
  <script src="_static/js/index.be7d3bbb2ef33a8344ce.js"></script>

  </body>
</html>